tests/Exceptionless.Tests/appsettings.yml

```diff
@@ -1,7 +1,11 @@
 ﻿---
 ConnectionStrings:
-  Redis.off: localhost,abortConnect=false
-  Elasticsearch.off: http://localhost:9200/
+  elasticsearch.off: server=http://localhost:9200
```

Comment:
I want to be able to just do a redis connectionstring and have it be used for all of the services that use redis if there isn't any overriding configuration for that specific service. For example, in our Azure environment, we would just configure Elasticsearch and Redis connection strings and then also provide a storage connection string that says to use Azure Blob Storage and then an additional queue connection string that say to use Azure Service Bus for queues. Everything else would default to using the redis connection string like the cache and messagebus. If we wanted to use Redis for queues as well then you could just remove the overriding queue connection string and it would fall back to using Redis. If no Redis connection string was defined then everything would fallback to using in memory. Basically, the idea is that you configure services that are available in the environment and we derive the best configuration from them unless you specifically override a service config and tell us exactly what to do.

Reply:
I agree and disagree. I think that we could do provider=xyz and then look for a connection string with that name and merge that connection string into the settings. I don't want a lot of configuration but I also want configuration to be explicit and not magic. I don't want to have to look at all the other options to see what's using Redis...